client/extensions/woocommerce/app/products/product-header.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import StickyPanel from 'components/sticky-panel';
+
+const ProductHeader = ( { onTrash, onDuplicate, onSave, translate } ) => {
+	const trashButton = onTrash &&
```

Comment:
Exactly. The callback function's presence indicates that the button should be shown. It's optional.